backend/db/seeds.rb

```diff
@@ -16,14 +16,13 @@
 category_habits = Category.create(title: 'Habits')
 
 challenge_gratitude_desc = <<~DESCRIPTION
-  While it’s important to analyze and learn from bad events, sometimes we can think too much about what goes wrong and
-  not enough about what goes right in our lives. A gratitude journal encourages ourselves to pay attention to the good
-  things in life we might otherwise take for granted. In that way, we start to become more attuned to the everyday
-  sources of pleasure around us—and the emotional tone of our life can shift in profound ways.
+  A gratitude journal encourages ourselves to pay attention to the good things in life we might otherwise take for granted.#{' 
```

Comment:
```suggestion
  A gratitude journal encourages ourselves to pay attention to the good things in life we might otherwise take for granted.
```

This isn't meant to be here, right?